### PR TITLE
Icon adjustments

### PR DESCRIPTION
**Pull request by Zed9876**
Some adjustments to the shelter and gate icons.

### CODE REVIEW

File: Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs [hunk 1]
```diff
@@ -85,6 +85,14 @@ public bool PlayerInShelter
             }
         }
 
+        public bool PlayerInAncientShelter
+        {
+            get
+            {
```
Comment (6fears7) [inline Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs:91]: May want to do either do a safety check in game to ensure non-MSC won't break here or wrap this in ModManager.MSC

**Review by 6fears7 — COMMENTED** [review]
If you could also submit a video of it in action that would be awesome. I trust you, I just don't trust myself to catch every piece of code

File: Story/OnlineUIComponents/OnlineHud.cs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿using HarmonyLib;
 using HUD;
 using System.Collections.Generic;
+using System.Diagnostics;
```
Comment (youbitchoc) [inline Story/OnlineUIComponents/OnlineHud.cs:4]: not needed?

File: Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs (hunk 1 above)
Comment (youbitchoc) [reply Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs:91]: imo it's safe as is, it's a bool that defaults to false

Comment (6fears7) [reply Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs:91]: Works for me

File: Story/OnlineUIComponents/OnlineHud.cs (hunk 2 above)
Comment (6fears7) [reply Story/OnlineUIComponents/OnlineHud.cs:4]: Probably just added when testing different methods to find the right one

Comment (6fears7) [reply Story/OnlineUIComponents/OnlineHud.cs:4]: @Zed9876 remove the newly added library and then merge. I'll pre approve

**Review by 6fears7 — APPROVED** [review]
Remove added diagnostics lib and will merge

**Comment by Zed9876** [comment]
Video of Icon adjustments:

https://www.youtube.com/watch?v=IRHGuOHZu34&ab_channel=JacobPreece

**Review by 6fears7 — APPROVED** [review]

**Pull request merged**